apps/demo-game/src/reducers/PeriodReducer.ts

```diff
@@ -1,57 +1,55 @@
-import { Action } from '@gbl-uzh/platform'
+import {
```

Comment:
should rename these files as they are no longer reducers ("service"?). maybe rename the folder reducers to services and the actionservice just contains a reducer, but a reducer can be part of a service :)